ydb/core/statistics/aggregator/tx_schemeshard_stats.cpp

```diff
@@ -21,18 +21,68 @@ struct TStatisticsAggregator::TTxSchemeShardStats : public TTxBase {
             << ", stats size# " << stats.size());
 
         NIceDb::TNiceDb db(txc.DB);
-        db.Table<Schema::BaseStatistics>().Key(schemeShardId).Update(
-            NIceDb::TUpdate<Schema::BaseStatistics::Stats>(stats));
 
-        Self->BaseStatistics[schemeShardId] = stats;
+        NKikimrStat::TSchemeShardStats statRecord;
+        Y_PROTOBUF_SUPPRESS_NODISCARD statRecord.ParseFromString(stats);
+
+        if (!Self->BaseStatistics.contains(schemeShardId) ||
+            !statRecord.HasAreAllStatsFull() || statRecord.GetAreAllStatsFull())
```

Comment:
Точно не пропущено отрицание? 

```suggestion
            !statRecord.HasAreAllStatsFull() || !statRecord.GetAreAllStatsFull())
```